modules/squaternion/readme.md

```diff
@@ -0,0 +1,32 @@
+# Simple Quaternions (`squaternion`)
+Vendored from [MomsFriendlyRobotCompany/squaternion](https://github.com/MomsFriendlyRobotCompany/squaternion) on 2023-01-16.
```

Comment:
Could we include the commit hash for the vendored revision?

Reply:
https://github.com/srobo/competition-simulator/pull/366/commits/e373663966baeb19ce3ab8c0a9526037fd9aada9